packages/react/src/file-input/FileInput.tsx

```diff
@@ -67,18 +68,22 @@ export const FileInput = forwardRef<HTMLInputElement, FileInputProps>(
 			[onChangeProp]
 		);
 
+		const fallbackHint =
+			hint || (accept && `Accepted files include: ${accept}`);
```

Comment:
Do this to match FileUpload
```suggestion
			hint || (accept && `Files accepted: ${accept}`);
```